docs/faq.rst

```diff
@@ -256,6 +256,18 @@ addresses of the users you wish to share with to give them permissions
 to your workflow. This is only available for hosted workflows and users
 with Google accounts linked to Terra.
 
+How do I add a specific version of a workflow to a collectionn and why would I want to?
```

Comment:
Typo: collection*

I think this a great description of why adding workflows to a collection is valuable, but glances over the 'how' thats indicated in the title. Perhaps a step-by-step list could be included?

Reply:
Changed title and changed picture on the page @NatalieEO referenced